launch_ros/launch_ros/actions/load_composable_nodes.py

```diff
@@ -39,7 +42,7 @@ def __init__(
         self,
         *,
         composable_node_descriptions: List[ComposableNode],
-        target_container: ComposableNodeContainer,
+        target_container: Union[SomeSubstitutionsType, ComposableNodeContainer],
```

Comment:
Actually, I'd rather have a single argument. In the most general case a user would have to provide a fully qualified node name (FQNN), but in Python we offer the chance to pass in a `ComposableNodeContainer` action from which an FQNN can be fetched. The later and current behavior is a bit of a heuristic, as the underlying node may ignore all name remappings.

Reply:
My personal thought when designing this was to reduce impact on current API, hence why a union type. I also started off thinking it should've been a separate argument. I agree that this makes the argument name vague with dual meaning, but I think this can be handled via better docstrings and docs in general.

Reply:
I like one argument that can either be a string (substitution) or a container node class. That way the two ways of specifying are mutually exclusive by design.

Reply:
ok, so one argument then.